sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -570,6 +572,7 @@ public virtual Response<RecognizeEntitiesResultCollection> RecognizeEntities(IEn
         public virtual async Task<Response<RecognizeEntitiesResultCollection>> RecognizeEntitiesAsync(IEnumerable<TextDocumentInput> inputs, TextAnalyticsRequestOptions options, CancellationToken cancellationToken = default)
         {
             Argument.AssertNotNull(inputs, nameof(inputs));
+            Argument.AssertNotNull(options, nameof(options));
```

Comment:
You're right there's a possible nullref here ... but since users may want to call the batch method without this parameter, why don't we default it and then create a new instance of it if it comes in null.

Reply:
yeah you right. No reason not to. 
I will make the change